config/configgrpc/configgrpc.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package configgrpc defines the gRPC configuration settings.
+package configgrpc
```

Comment:
Does it have to be in a new package? Can it be included in https://github.com/open-telemetry/opentelemetry-service/blob/master/config/configmodels/configmodels.go?

Reply:
I like a separate package so components that don't have gRPC configuration don't have that exposed in under the configmodel package. That said it is not like configmodel is a large package with many definitions. Let's see if there are other opinions.